xbmc/settings/DiscSettings.cpp

```diff
@@ -58,12 +59,12 @@ CDiscSettings& CDiscSettings::GetInstance()
 
 void CDiscSettings::OnSettingChanged(std::shared_ptr<const CSetting> setting)
 {
-#ifdef TARGET_WINDOWS
   if (setting == NULL)
     return;
 
   const std::string &settingId = setting->GetId();
 
+#if (BLURAY_VERSION >= BLURAY_VERSION_CODE(1,0,1))
```

Comment:
@ace20022 I can do, but shouldn't the ifdef only enclose as little as actually necessary?
If one adds another setting to DiscSettings later the setting null check would also apply to the new one.